CONTRIBUTING.md

```diff
@@ -228,16 +228,17 @@ Run the following release commands, replacing `7.0.0` with your version number:
   - Then reset by deleting your tag and branch locally and remotely.
     - `git push --delete origin v7.1.0`
     - `git tag --delete v7.1.0`
-    - `git push -d origin xx/release-7.0.0`
-    - `git branch -d xx/release-7.0.0`
+    - `git push -d origin release-7.0.0`
+    - `git branch -d release-7.0.0`
   - Now redo the steps above starting with creating "A clean local copy of the Turf Github repository"
 
 - If the release action was successful, now create a Pull Request for the release to update master.
   - You may be given a link in the output of the branch/tag push command to create the PR.
   - If you don't get this message, just go to https://github.com/Turfjs/turf/pulls and you should be prompted at the top to create a PR for this new branch you just pushed.
   - If that prompt doesn't appear, then just create a new pull request from the PR page and make sure the title is the version number e.g. `v7.0.0` and that it is merging your release branch -> to master.
   - Here is an example PR - https://github.com/Turfjs/turf/pull/2615.
-  - Get approval for the release PR and merge it!
+  - Get approval for the release PR, then "Squash and merge" it
+  - Click "Restore branch".  Github will have auto-deleted the release branch on merge.  Restoring is required because the release tag points at this release branch.
```

Comment:
Is there anything we can do to stop this happening? I don't recall seeing this when I squash+merge other PRs.

Reply:
My mistake, the auto-delete setting is not on for this repository (it's a top-level setting on the settings page).  I must have clicked to delete the branch after merge then.  I'll change the text accordingly.